explorer/site-config/hca-dcp/dev/index/filesEntityConfig.ts

```diff
@@ -27,6 +28,14 @@ export const filesEntityConfig: EntityConfig<FilesResponse> = {
   label: "Files",
   list: {
     columns: [
+      {
+        componentConfig: {
+          component: Components.AzulFileDownload,
+          viewBuilder: buildFileDownload,
+        } as ComponentConfig<typeof Components.AzulFileDownload>,
+        header: " ",
```

Comment:
There's an issue here. If we try to make a column with an empty header we get an error on the table's creation. I didn't did a lot of search about it, fixing it might need some deep research about how react-table headers creation work. A quick workaround is to make the header a space.

Reply:
Thanks for the background, Bruce!